lib/src/DateRangePicker/DateRangePickerViewDesktop.tsx

```diff
@@ -1,9 +1,9 @@
 import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
 import { DateRange } from './RangeTypes';
 import { useUtils } from '../_shared/hooks/useUtils';
-import { makeStyles } from '@material-ui/core/styles';
 import { calculateRangePreview } from './date-range-manager';
-import { Calendar, CalendarProps } from '../views/Calendar/Calendar';
+import Calendar, { CalendarProps } from '../views/Calendar/Calendar';
```

Comment:
That's what I am talking about
```js
import { Calendar, CalendarProps } from '...'
```

looks much better

Reply:
For context, the reason I went for these changes is because eslint was complaining about duplicate exports (default and named), I went on consolidating with what was most popular in the codebase.